versions.html

```diff
@@ -20,6 +20,9 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="master/">master (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.10.0/">v0.10.0 (release candidate)</a>
```

Comment:
Since we are very close to release, you might want to remove `stable release` label from `0.9.0` and add it to `0.10.0`. Otherwise you will have to make a separate PR.